srcpkgs/python3-txtorcon/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'python3-txtorcon'
 pkgname=python3-txtorcon
-version=21.1.0
-revision=4
+version=22.0.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3 python3-setuptools"
 depends="lsof python3-automat python3-cryptography python3-six python3-Twisted
- python3-zope.interface"
+ python3-zope.interface which"
 checkdepends="${depends} python3-mock python3-pytest"
 short_desc="Twisted-based asynchronous Tor control protocol implementation"
 maintainer="travankor <travankor@tuta.io>"
 license="MIT"
 homepage="https://txtorcon.readthedocs.io/en/latest/"
+changelog="https://raw.githubusercontent.com/meejah/txtorcon/main/docs/releases.rst"
 distfiles="https://github.com/meejah/txtorcon/archive/v${version}.tar.gz"
-checksum=61ef40e110a2b52255a6033293d02647dedb63ed9311ce572571c777b2302750
-make_check=ci-skip
+checksum=195c2e3d1ed4ad564570d20af54812163ca1de9e9ee034f0c8276fd4410759e7
+make_check=ci-skip # Tests run locally but fail on CI
```

Comment:
make_check=ci-skip already tells us tests run locally but fail in CI, so the comment is redundant, it should probably be something like "can not open ports in CI"